src/ASM/LR/ASMu2Drecovery.C

```diff
@@ -426,7 +428,8 @@ LR::LRSplineSurface* ASMu2D::regularInterpolation (const RealArray& upar,
     // optimization note:
     // without an element id, splineValues will be stored as a full dense vector
     for (size_t j = 0; j < nBasis; j++)
-      A(i+1,j+1) = splineValues.basisValues[j];
+      if (utl::trunc(splineValues.basisValues[j]) != 0.0)
```

Comment:
utl::trunc var vel i utganspunktet kun ment for bruk ifm. (debug) utskrift av flyttalls-størrelser, som skriver ut 0.0 hvis lavere enn en gitt toleranse (som kan settes via en variabel). Å bruke den slik for numerisk oppførsel er litt skummelt, hvis denne toleransen er satt noe slakk. Da vil resultatet kunne blir påvirka utilsektet. Bør vel i dette tilfellet heller finne en mer robust løsning for å avgjøre hvilke basis funksjoner som er logisk lik 0.0